src/components/views/dialogs/InviteDialog.js

```diff
@@ -411,6 +411,58 @@ export default class InviteDialog extends React.PureComponent {
             return scores;
         }, {});
 
+        // Now that we have scores for being in rooms, boost those people who have sent messages
+        // recently, as a way to improve the quality of suggestions. We do this by checking every
+        // room to see who has sent a message in the last few hours, and giving them a score
+        // which correlates to the freshness of their message. In theory, this results in suggestions
+        // which are closer to "continue this conversation" rather than "this person exists".
+        const trueJoinedRooms = client.getRooms().filter(r => r.getMyMembership() === 'join');
+        const now = (new Date()).getTime();
+        const maxAgeConsidered = now - (60 * 60 * 1000); // 1 hour ago
```

Comment:
this is a time rather than a duration – perhaps rename to `earliestAgeConsidered`?